apps/gjenlevendepensjon-ui/src/locales/en.ts

```diff
@@ -193,6 +193,9 @@ export default {
     "omDegOgAvdoed.avdoed.fornavn": "First name(s)",
     "omDegOgAvdoed.avdoed.etternavn": "Family name",
     "omDegOgAvdoed.avdoed.dato": "Date",
+    "omDegOgAvdoed.avdoed.dato.etterNovember": "Because the deceased's time of death was after November 2023 you must apply for a ",
+    "omDegOgAvdoed.avdoed.dato.etterNovember.link": "survivor's pension.",
```

Comment:
Her skal vell engelsk overettelse for omstillingsstønad?
`adjustment allowance`

Reply:
Fix'd!